.env.local.template

```
@@ -1,2 +1,3 @@
 NODE_ENV=development
-MONGO_URI='mongodb://<user>:<pass>@<host>:<port>/<database>?<connection options>'
\ No newline at end of file
+MONGO_URI='mongodb://<user>:<pass>@<host>:<port>/<database>?<connection options>'
+MAPBOX_TOKEN=[ENTER_MAP_API_KEY]
```

This needs updating as it's no longer correct and will cause confusion
```suggestion
NEXT_PUBLIC_MAPBOX_TOKEN=[ENTER_MAP_API_KEY]
```